package.json

```diff
@@ -78,8 +79,10 @@
     "fixtures:load": "cozy-fixtures load tests/fixtures",
     "fixtures:loader": "coffee tests/fixtures/loader.coffee",
     "lint": "npm-run-all 'lint:*'",
-    "lint:coffeelint": "coffeelint --file coffeelint.json --quiet ./server/**/*.coffee",
-    "lint:jshint": "coffee-jshint --options node ./server/**/*.coffee",
+    "lint:server": "npm-run-all 'lint:server:*:*'",
+    "lint:server:code:coffeelint": "./node_modules/.bin/coffeelint --file coffeelint.json  server.coffee ./server/*.coffee ./server/**/*.coffee",
```

Comment:
You shouldn't explicitely use `./node_modules/.bin` prefix: npm scripts add it to the PATH env, so directly call `coffeelint` in script will make npm use primarily the locally installed package :wink:.